ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-commands.sh

```diff
@@ -14,31 +14,26 @@ chmod +x ${YQ}
 # Dynamically get CNV catalog image that was provided to the job via gangway API
 CNV_PRERELEASE_CATALOG_IMAGE=$(curl -s https://prow.ci.openshift.org/prowjob?prowjob="${PROW_JOB_ID}" |\
   ${YQ} e '.spec.pod_spec.containers[0].env[] | select(.name == "CNV_PRERELEASE_CATALOG_IMAGE") | .value')
+CNV_SUBSCRIPTION_SOURCE=${CNV_SUBSCRIPTION_SOURCE:-cnv-prerelease-catalog-source}
+CNV_PRERELEASE_VERSION=${CNV_PRERELEASE_VERSION:-$(ocp_version)}
 
-if [ -n "${CNV_PRERELEASE_CATALOG_IMAGE}" ]
+if [ -n "${CNV_PRERELEASE_CATALOG_IMAGE}" -o -n  "${CNV_SUBSCRIPTION_SOURCE}" ]
```

Comment:
The logic both before and after your change is difficult for me to completely follow.

From what I gather, we have 3 potential paths.
1. custom catalog image from brew. In this scenario we want to default to `stable` release channel i think.
2. using nightly. in this scenario we want to default to `nightly-$ocp_version` release channel?
3. and finally installing cnv directly the public `redhat-operators` subscription source, where we want to default to stable as well ?

This line below seems like it's resulting in CNV_RELEASE_CHANNEL always being set to `stable` though because `CNV_SUBSCRIPTION_SOURCE` env var will always be set to a default value. 
```
if [ -n "${CNV_PRERELEASE_CATALOG_IMAGE}" -o -n "${CNV_SUBSCRIPTION_SOURCE}" ]
```

Reply:
I will go back to spliet the if statement on three.